base/div.jl

```diff
@@ -3,12 +3,11 @@
 """
     div(x, y, r::RoundingMode=RoundToZero)
 
-Compute the remainder of `x` after integer division by `y`, with the quotient rounded
-according to the rounding mode `r`. In other words, the quantity
+The quotient from Euclidean division. Computes x/y, rounded to an integer according
+to the rounding mode `r`. In other words, the following quantity:
 
-    y*round(x/y,r)
+    round(x/y,r)
 
-without any intermediate rounding.
```

Comment:
the without any intermediate rounding is still applicable (there'd be rounding in the `x/y` if done in floating point).

Reply:
Ah, yes, good call! Fixed. :)